cdap-spark-core/src/main/java/co/cask/cdap/app/runtime/spark/SparkRunnerClassLoader.java

```diff
@@ -111,10 +117,14 @@
       } else if (name.equals(SPARK_STREAMING_CONTEXT_TYPE.getClassName())) {
         // Define the StreamingContext class by rewriting the constructor
         cls = defineContext(SPARK_STREAMING_CONTEXT_TYPE, name, is);
+      } else if (name.equals(AKKA_REMOTING_TYPE.getClassName())) {
+        // Define the akka.remote.Remoting to write using of scala.concurrent.ExecutionContext.Implicits.global
```

Comment:
fixed. Also moved the comment to the javadoc of the `defineAkkaRemoting` method.